vendor/mruby/Makefile.am

```diff
@@ -37,6 +37,9 @@ endif
 
 include sources.am
 include built_sources.am
+if PLATFORM_WIN32
+BUILT_SOURCES += mruby-dir/src/Win/dirent.c
```

Comment:
`src/Win/dirent.c` is included in `src/dir.c`: https://github.com/iij/mruby-dir/blob/master/src/dir.c#L19
So we don't need to build it manually.

We need to just bundle it into our tar.gz.